preditor/gui/ui/loggerwindow.ui

```diff
@@ -1013,6 +1002,17 @@ at the indicated line in the specified text editor.
     <string>Visually Separate PrEditor Traceback</string>
    </property>
   </action>
+  <action name="uiRunFirstWorkboxACT">
+   <property name="text">
+    <string>Run First Workbox</string>
+   </property>
+   <property name="toolTip">
+    <string>Run First Workbox, which may have things like common import, helper functions, etc.</string>
```

Comment:
It would probably be good to somehow clarify that its running the first workbox in the first group not the current group, at least in the tooltip.

This is a nice feature, but I was very cautious to run this the first time because many of my tabs contain potentially dangerous code if the full tab is run accidentally.

Reply:
Nope, its up to the user to not put dangerous code there but this way they know exactly where that is. Looks good, thanks.